samples/node/src/jsonid-adapter.ts

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright IBM Corp. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+import { Identity, Signer, signers } from 'fabric-gateway';
+import fs from 'fs/promises';
+import path from 'path';
+import crypto from 'crypto';
```

Comment:
Sometimes I see TypeScript flag up errors for this import style, either depending on the tsconfig or the type of module being imported. I find the `import { promises as fs } from 'fs'` style typically causes fewer issues, but as long as it's building and working, whatever format you prefer is fine by me.